service/src/main/java/tds/assessment/services/impl/AssessmentAssembler.java

```diff
@@ -0,0 +1,69 @@
+package tds.assessment.services.impl;
```

Comment:
Why is this pulled out of the service?  I'm having trouble seeing the benefit since this is only ever used within the AssessmentService.

I understand we chatted about this yesterday but starting to question my comments after seeing it coded.